README.md

```diff
@@ -64,8 +66,10 @@ If you want to run preprocessing yourself:
 libsox-fmt-ffmpeg or [installing from
 source](https://github.com/neuralaudio/hear-eval-kit/issues/156#issuecomment-893151305).
 
-This will take about 2 user-CPU-hours for the open tasks. 150 GB free
-disk space is required while processing. Final output is 11 GB.
+This will take about 2 user-CPU-hours for the open tasks, when using
+mode 'default'. 150 GB free disk space is required while processing.
+Final output is 11 GB. Much more time and disk space is required
+for 'all'.
```

Comment:
We're now using 'all' for two args (task selection and the mode) -- let's make it really clear which one we are referring to here.